src/lib.c

```diff
@@ -455,10 +480,42 @@ void calc_per_store(cart_item_s cart_item[], int n_shopping_list, int n_stores,
 
     qsort(cart, n_stores, sizeof(cart_sum_s), compare_cart);
```

Comment:
Vi ku udregne den totale sum med gas før qsort i stedet for i print functionen, så den sortere efter den totale sum med gas i stedet for de billigste varer?